src/main/java/com/vcsajen/yourcustompaintings/YourCustomPaintings.java

```diff
@@ -85,7 +85,12 @@
  * Main plugin class
  * Created by VcSaJen on 26.07.2017 17:21.
  */
-@Plugin(id = "yourcustompaintings", name = "YourCustomPaintings", description = "Upload your own custom paintings to minecraft server!")
+@Plugin(
+        id = "yourcustompaintings",
+        name = "YourCustomPaintings",
+        description = "Upload your own custom paintings to minecraft server!",
+        version = "1.1"
```

Comment:
I do not accept pull requests at this point of time, sorry.
P.S. Version is auto-generated with SpongeGradle. This line is unnecessary and breaks DRY.